lib/routes/charge/sale_view.dart

```diff
@@ -235,6 +226,49 @@ class SaleViewState extends State<SaleView> {
           );
         });
   }
+
+  _buildActions(AccountModel account) {
+    if (account != null) {
```

Comment:
Doesn't seem to be an intuitive way to distinguish between read only and in progress sales view. No reason it shouldn't be explicit.
I suggest to separate to two functions one for readonly and one for the editable view.

Reply:
[Done](https://github.com/breez/breezmobile/pull/405/commits/68e5e421f479aac77208714bcb43fb65226b4e2b)